docs/src/functions/engineering/erf-precise.md

```diff
@@ -3,9 +3,50 @@ layout: doc
 outline: deep
 lang: en-US
 ---
+# ERF.PRECISE function
```

Comment:
I think ERF.PRECISE should point to the same page as ERF and add a note that ERF. PRECISE is there only for comptibility reasons.